src/Compilers/Core/CodeAnalysisTest/DefaultAnalyzerAssemblyLoaderTests.cs

```diff
@@ -30,63 +30,131 @@
 
 namespace Microsoft.CodeAnalysis.UnitTests
 {
-    [CollectionDefinition(Name)]
-    public class AssemblyLoadTestFixtureCollection : ICollectionFixture<AssemblyLoadTestFixture>
-    {
-        public const string Name = nameof(AssemblyLoadTestFixtureCollection);
-        private AssemblyLoadTestFixtureCollection() { }
-    }
+#if NETCOREAPP
 
-    public sealed class DefaultAnalyzerAssemblyLoaderTests : TestBase
+    public sealed class InvokeUtil
     {
-#if NETCOREAPP
-        private sealed class InvokeUtil
+        public void Exec(Action<string> testOutputHelper, AssemblyLoadContext alc, bool shadowLoad, string typeName, string methodName)
```

Comment:
Consider naming this something like `compilerContext`

Reply:
cleaned this up in the next PR where i fixed up names quite a bit.